pyproject.toml

```diff
@@ -68,6 +68,13 @@ archs = ["auto", "aarch64"]
 [tool.cibuildwheel.macos]
 archs = ["x86_64", "universal2", "arm64"]
 
+[tool.cibuildwheel.windows]
+archs = ["AMD64"]
+test-command = [
+  "cd /d {package}\tests",
```

Comment:
The `/d` option must be specified or this error message gets reported: `The system cannot find the path specified.`

Reference: https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/cd#:~:text=Description-,/d,-Changes%20the%20current